main.cpp

```diff
@@ -257,6 +259,11 @@ int main(void) {
       default:
         break;
       }
+#ifdef NXDK
+      // Let's not hog CPU for nothing.
+      XVideoWaitForVBlank();
+      SwitchToThread();
```

Comment:
Why can't we just call SwitchToThread directly? Why waiting for the VBlank?

Reply:
This was answered on the XboxDev Discord server, but I'll relay it here for completeness;
Plan is to move away from SDL2 as a renderer and do something more fancy in the future. As transfer speeds don't seem to have been negatively effected by this change, it might as well make its way into here sooner rather than later.